### DISCUSSION TITLE
AutoLayout doesn't work in WinUI

### DISCUSSION THREAD

**Discussion by ttustonic** [unoplatform/uno#8967] [Q&A] [answered]
### Current behavior

I'm trying a simple AutoLayout example from: https://playground.platform.uno/#848c7509 in WinUI application, but it only shows a blank page.

### Expected behavior

It should display something similar to the sample from the above url.

### How to reproduce it (as minimally and precisely as possible)

Create a new project:
`dotnet new unoapp-winui -o testautolayout`
add Nuget packages (tried both vers 2.0.0 and a latest dev 2.1.0-dev.5), set MainPage.xaml content to:
```
<StackPanel>
        <utu:AutoLayout Width="100" Height="100" 
                    Orientation="Horizontal" 
                    Margin="25" 
                    Background="Lime"
                    PrimaryAxisAlignment="Start">

            <Border Background="Coral"
                        utu:AutoLayout.PrimaryLength="100"
                        utu:AutoLayout.CounterLength="100">
                <TextBlock>Start</TextBlock>
            </Border>
        </utu:AutoLayout>
    </StackPanel>
```

and run a Windows.Package project
[testautolayout.zip](https://github.com/unoplatform/uno/files/8834344/testautolayout.zip)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report!

In order to use the Toolkit, you'll need to add at least the following to your `app.xaml` file:
```xaml
<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
```

You can find the [full documentation here](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/getting-started.html).

**Comment by ttustonic** [comment]
That's it!
Thanks, Tom
